INSTALL

```diff
@@ -479,8 +479,7 @@
   enable-tls1_3
                    TODO(TLS1.3): Make this enabled by default
                    Build support for TLS1.3. Note: This is a WIP feature and
-                   does not currently interoperate with other TLS1.3
-                   implementations! Use with caution!!
+                   may have nasty bugs.  Use with caution!!
```

Comment:
Losing the interoperability text entirely is possibly not quite right. Currently we don't interoperate with a whole bunch of stuff still on draft-18.